samples/set-reference-scale/src/main/java/com/esri/arcgismaps/sample/setreferencescale/screens/SetReferenceScaleScreen.kt

```diff
@@ -0,0 +1,280 @@
+ */
+@Composable
+fun SettingsControls(
+    referenceScaleOptions: List<Double>,
+    selectedReferenceScale: Double,
+    onReferenceScaleSelected: (Int) -> Unit,
+    onShowLayers: () -> Unit,
+    currentMapScale: Double,
+    onSetToReferenceScale: () -> Unit
+) {
+    // A Left column of hints and right of controls
+    Card(Modifier.padding(12.dp)) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(12.dp),
+            verticalArrangement = Arrangement.spacedBy(12.dp)
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(text = "Map's reference scale:")
+                DropDownMenuBox(
+                    modifier = Modifier.padding(start = 12.dp),
+                    textFieldValue = "1:${selectedReferenceScale.toInt()}",
+                    textFieldLabel = "Reference Scale",
+                    dropDownItemList = referenceScaleOptions.map { "1:${it.toInt()}" },
+                    onIndexSelected = onReferenceScaleSelected
+                )
+            }
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(text = "Toggle layer visibility:")
+                Button(onClick = onShowLayers) {
+                    Text(text = "Layers")
+                }
+            }
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(text = "Map Scale: 1:${if (currentMapScale.isNaN()) "..." else currentMapScale.toInt()}")
+                Button(
+                    onClick = onSetToReferenceScale,
+                    enabled = !currentMapScale.isNaN() && (currentMapScale != selectedReferenceScale)
+                ) {
+                    Text(text = "Set to reference scale")
+                }
+            }
+        }
+    }
+}
+
+/**
+ * Dialog composable that displays a list of operational [layers] and a toggle for
+ * each layer's scalesSymbols property.
+ */
+@Composable
+fun LayersDialogContent(
+    layers: List<LayerToggleState>,
+    onToggleLayerScales: (LayerToggleState, Boolean) -> Unit,
+    onDone: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(12.dp),
+        horizontalAlignment = Alignment.End,
+        verticalArrangement = Arrangement.spacedBy(12.dp)
+    ) {
+        Text(
+            modifier = Modifier.fillMaxWidth(),
+            text = "Layers",
```

Comment:
The title suggest we are toggling the layer visibility, suggestion to make it more descriptive
```suggestion
            text = "Layer Symbol Scaling",
```